public/assets/html5shiv-6152e82d89cb5429c94bca671535b4d374aa465dd0543b6a24848e964d14fde7.js

```diff
@@ -0,0 +1,327 @@
+/**
+* @preserve HTML5 Shiv 3.7.3 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
```

Comment:
A bunch of the files can get added to `.gitignore` from the `public/assets` directory. I _think_ this might be the thing to add: `public/assets/**`